packages/perseus/src/widgets/radio/choice-icon.jsx

```diff
@@ -57,23 +57,28 @@ class SATChoiceIcon extends React.Component<{
             color = styleConstants.white;
             backgroundColor = styleConstants.satBlue;
         }
-        return {color, backgroundColor, borderColor};
+
+        let borderRadius;
+        if (multipleSelect) {
+            borderRadius = 3;
+        } else {
+            borderRadius = SAT_ICON_SIZE;
+        }
+
+        return {color, backgroundColor, borderColor, borderRadius};
     }
 
     render(): React.Node {
-        const {letter, a11yText, reviewMode, checked, correct, crossedOut} =
-            this.props;
-        const {color, backgroundColor, borderColor} = this.constructStyles(
-            reviewMode,
-            correct,
-            checked,
-        );
+        const {letter, a11yText, crossedOut} = this.props;
+        const {color, backgroundColor, borderColor, borderRadius} =
+            this.constructStyles();
 
         return (
             <div className={css(styles.iconWrapper)}>
                 <div
                     className={css(styles.satCircle)}
-                    style={{backgroundColor, borderColor}}
+                    data-test-id="choice-icon__sat-choice-icon"
```

Comment:
suggestion: I think we should be avoid hard-coding data-test-ids into the components themselves. Instead of using data-test-id in tests to find components, it's preferable to use ARIA roles instead. Would it be possible to set the appropriate role here and check for that in the tests you wrote?